### PR TITLE
MM-46653: Decrease the height of horizontal rules

### PR DESCRIPTION
**Pull request by sinansonmez**
#### Summary
This PR changes the height of horizontal rule from `4px` to `2px`

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/20917
JIRA: https://mattermost.atlassian.net/browse/MM-46653

#### Screenshots

|  before  |  after  |
|----|----|
| Screen Shot 2022-08-31 at 00 07 43 | Screen Shot 2022-08-31 at 00 10 06 |

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by AshishDhama — APPROVED** [review]

**Comment by sinansonmez** [comment]
Kindly pinging for review @michaelgamble

**Review by michaelgamble — CHANGES_REQUESTED** [review]
After reading the original Jira ticket I know it was phrased very subjectively as in "It should be 1px or 2px... ".. So I did a little digging and realized that we really should create consistency with how it's being rendered in the post input field render preview.. i.e. what I preview is what I get...

For reference this is what it looks like:
image

As you can see it's 1px in height.. also we should align the color..

To make it consistent we would need the following changes:
- 1px in height
- color should be center channel text @ 20% opacity

**Comment by sinansonmez** [comment]
Thanks for the feedback @michaelgamble, updated as suggested

File: sass/layout/_markdown.scss [hunk 1]
```diff
@@ -330,12 +330,11 @@ h6.markdown__heading {
...
 .post__body {
     hr {
-        height: 4px;
+        height: 1px;
         padding: 0;
         border: 0 none;
         margin: 15px 0 16px;
-        background-color: #e7e7e7;
-        opacity: 0.2;
+        background-color: rgba(var(--center-channel-text), 0.2) !important;
```
Comment (AshishDhama) [inline sass/layout/_markdown.scss:337]: @sinansonmez is it necessary to use Important here, Would be great if we could avoid by using some class combination with more specificity.

Comment (sinansonmez) [reply sass/layout/_markdown.scss:337]: I also didn't feel comfortable to put `!important`. That is why also I re-requested review from you 😄 
However current `background-color` is coming from `inline` styling most probably from the line below. 

I will do a deep dive to do it without `!important`

https://github.com/mattermost/mattermost-webapp/blob/2719a868113c3db10f2beb7b701552f7e7043a8d/utils/utils.tsx#L423

Comment (michaelgamble) [reply sass/layout/_markdown.scss:337]: also a quick side note, im not sure why but its rendering out as (0,0,0,0) for some reason... but I agree we should be avoiding the use of an !important tag
image

(And I don't mean to be fussy the opacity should be .24)

Comment (sinansonmez) [reply sass/layout/_markdown.scss]: Now it should be fixed without using `important`

**Review by AshishDhama — APPROVED** [review]

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by michaelgamble — APPROVED** [review]
LGTM

**Review by jgilliam17 — APPROVED** [review]
Thanks @sinansonmez 
Tested, looks good to merge.
Verified height of horizontal rule is ~not~ now 1px and using center channel text color at 24% opacity.
[E2E report ](https://mattermost-cypress-report.s3.amazonaws.com/245188-e46bac0-onprem-ent-webapp-pr-11029-mm-ee-test:e46bac0/mochawesome.html)- no PR related failures.

**Pull request merged**